.github/workflows/build-wheels.yml

```diff
@@ -32,22 +32,23 @@ jobs:
         with:
           fetch-depth: 0
       - name: setup rust
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
         with:
-          profile: minimal
           toolchain: stable
-          default: true
           target: ${{ matrix.rust-target }}
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
           python-version: "3.10"
-      - name: build wheel
+
+      - name: install dependencies
+        run: python -m pip install wheel
+      - name: build equistore-core wheel
         env:
           RUST_BUILD_TARGET: ${{ matrix.rust-target }}
         run: |
-          python -m pip install wheel
-          python setup.py bdist_wheel --plat-name ${{ matrix.platform-name }}
+          cd python/equistore-core
+          python setup.py bdist_wheel --plat-name ${{ matrix.platform-name }} --dist-dir ../../dist/
```

Comment:
and why are we not using tox for building the binary here. Seems like duplicated.